content/resources/faq/_index.md

```diff
@@ -101,69 +101,69 @@ as a template for your own `gitops.json` file. Set navigation title and login
 title at your choice. Then reroll portal with `gen3 kube-setup-portal`.
 
 
-**I updated Fence configuration but changes don't appear?**
+<h4>I updated Fence configuration but changes don't appear?</h4>
 
 After updating the Fence configuration, run `kubectl delete secret
 fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
 to take effect with `gen3 roll fence`.
 
 
-**GraphiQL on Query page form throwing errors.**
+<h4>GraphiQL on Query page form throwing errors.</h4>
 
 You may need to hit the `Switch to Graph Model` button in the upper right
 corner.
 
 
-**How to run SNS service?**
+<h4>How to run SNS service?</h4>
 
 Please refer to [Cloud
 Automation](https://github.com/uc-cdis/cloud-automation/tree/master/tf_files/aws/modules/data-bucket-queue)
 documentation.
 
 
-**Arborist and Sheepdog don't work as expected, what should I check?**
+<h4>Arborist and Sheepdog don't work as expected, what should I check?</h4>
 
 `kubectl logs <name of pod>` can show you corresponding errors in the logs.
 
 
-**I uploaded the new dictionary, what service should I restart?**
+<h4>I uploaded the new dictionary, what service should I restart?</h4>
 
 First, restart Peregrine `gen3 roll peregrine` and Sheepdog `gen3 roll
 sheepdog`.  After these two services restarted, please portal with `gen3 roll
 portal`.  Please note, if you are making significant changes to the dictionary,
 you will also need to update `portal_app` in the manifest.
 
 
-**I branched out windmill portal, should I update manifest with a new version of
-portal?**
+<h4>I branched out windmill portal, should I update manifest with a new version of
+portal?</h4>
 
```

Comment:
I'd recommend putting "the" between "out" and "windmill".  And capitalize "Windmill"

Reply:
Fixed in 4b5ddf4